modules/swagger-codegen/src/main/java/io/swagger/codegen/examples/ExampleGenerator.java

```diff
@@ -20,71 +22,107 @@
 import io.swagger.models.properties.StringProperty;
 import io.swagger.models.properties.UUIDProperty;
 import io.swagger.util.Json;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.math.BigDecimal;
 import java.util.ArrayList;
-import java.util.Arrays;
+import java.util.Collections;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
 public class ExampleGenerator {
+    private static final Logger log = LoggerFactory.getLogger(ExampleGenerator.class);
```

Comment:
@bartkummel thanks for the PR. Just a minor suggestion. We use `logger` instead of `log` in other Java class files. Do you mind using the same (ie. `logger`) to make it consistent with other files?